frontend/pages/queries/QueryPage/screens/QueryEditor.tsx

```diff
@@ -77,10 +82,10 @@ const QueryEditor = ({
     [key: string]: string;
   }>({});
 
-  const onSaveQueryFormSubmit = debounce(async (formData: IQueryFormData) => {
+  const saveQuery = debounce(async (formData: ICreateQueryRequestBody) => {
```

Comment:
Our frontend has a pattern of `onSubmit` or `onSave`, I don't know how I feel about just `saveQuery`, we can chat about it more in a frontend sync but I don't think this naming convention should hold up the approval

Reply:
This is to distinguish this function from the `onClickSaveQuery` function that calls it a couple levels down in `QueryForm` > `SaveQueryModal`